src/main/webapp/content/files/precisionOncologyTx/presitionOncologyTx.json

```diff
@@ -0,0 +1,850 @@
+﻿[
+  {
+    "id": 1,
+    "year": "1998 – 2001*",
+    "tx": "Arsenic trioxide",
+    "biomarker": "PML-RARɑ Fusion",
+    "biomarkerDetection": "DNA/NGS-based detection or FISH",
+    "drugClassification": "Follow-on"
+  },
+  {
+    "id": 2,
+    "year": "1998 – 2001*",
+    "tx": "Denileukin diftitox",
+    "biomarker": "CD25+",
+    "biomarkerDetection": "IHC",
+    "drugClassification": "NA"
+  },
+  {
+    "id": 3,
+    "year": "1998 – 2001*",
+    "tx": "Exemestane",
+    "biomarker": "ER+",
+    "biomarkerDetection": "IHC",
+    "drugClassification": "NA"
+  },
+  {
+    "id": 4,
+    "year": "1998 – 2001*",
+    "tx": "Gemtuzumab ozogamicin",
+    "biomarker": "CD33+",
+    "biomarkerDetection": "Flow cytometry",
+    "drugClassification": "NA"
+  },
+  {
+    "id": 5,
+    "year": "1998 – 2001*",
+    "tx": "Imatinib",
+    "biomarker": "BCR-ABL1 Fusion (Ph+)",
+    "biomarkerDetection": "DNA/NGS-based detection or FISH",
+    "drugClassification": "First-in-class"
+  },
+  {
+    "id": 6,
+    "year": "1999 – 2001*",
+    "tx": "Imatinib",
+    "biomarker": "KIT+",
+    "biomarkerDetection": "IHC; DNA/NGS-based detection can be used",
+    "drugClassification": "First-in-class"
+  },
+  {
+    "id": 7,
+    "year": "2000 – 2001*",
+    "tx": "Imatinib",
+    "biomarker": "PDGFRA/B Fusions",
+    "biomarkerDetection": "DNA/NGS-based detection",
+    "drugClassification": "First-in-class"
+  },
+  {
+    "id": 8,
+    "year": "2001 – 2001*",
+    "tx": "Imatinib",
+    "biomarker": "PDGFB Fusions",
+    "biomarkerDetection": "DNA/NGS-based detection",
+    "drugClassification": "First-in-class"
+  },
+  {
+    "id": 9,
+    "year": "2002 – 2001*",
```

Comment:
Should this be 2001-2002?

Reply:
@calvinlu3 don't worry about the content. I will update once the curation team finalize it.